src/instance.ts

```diff
@@ -192,6 +238,249 @@ class Instance extends common.GrpcServiceObject {
     this.databases_ = new Map<string, Database>();
   }
 
+  /**
+   * Get a reference to a Backup object.
+   *
+   * @throws {Error} If any parameter is not provided.
+   *
+   * @param {string} backupId The name of the backup.
+   * @param {string} databasePath the path of the backup.  Only needs to be specified for new backups.
+   * @param {PreciseDate} expireTime expiry time of the backup.  Only needs to be specified for new backups.
+   * @return {Backup} A Backup object.
+   *
+   * @example
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = new Spanner();
+   * const instance = spanner.instance('my-instance');
+   * const backupExpiryDate = new PreciseDate(Date.now() + 1000 * 60 * 60 * 24)
+   * const backup = instance.backup('my-backup', 'projects/my-project/instances/my-instance/databases/my-database', backupExpiryDate);
+   */
+  backup(
+    backupId: string,
+    databasePath: string,
+    expireTime: PreciseDate
+  ): Backup {
+    if (!backupId) {
+      throw new Error('A backup ID is required to create a backup.');
```

Comment:
This is kind of silly, but if you capitalize `...to create a Backup`, it could help imply we're talking about a "Backup" object as opposed to a remote "backup" resource.

Reply:
I see what you mean. Fixed.